### PR TITLE
Update items

### PR DESCRIPTION
**Pull request by ENORMOUZ**

### CODE REVIEW

**Comment by ENORMOUZ** [comment]
funny how its called Archeologist in bestiary but Archaeologist in wikis

**Comment by ENORMOUZ** [comment]
Remind me to add Byzantium Dye to Voidling drop table

**Comment by ENORMOUZ** [comment]
Nvm it only drops from minion spawned voidlings and not the guys in end (i think)

File: items/NECRON_BOSS.json [hunk 1]
```diff
@@ -272,7 +272,7 @@
           ]
         },
         {
-          "id": "",
```
Comment (jani270) [inline items/NECRON_BOSS.json:275]: Fixed in another pr

**Review by jani270 — CHANGES_REQUESTED** [review]

File: items/NECRON_BOSS.json (hunk 1 above)
Comment (ENORMOUZ) [reply items/NECRON_BOSS.json:275]: how do i unfix my version or smth

Comment (jani270) [reply items/NECRON_BOSS.json:275]: Revert your commit?

**Comment by ENORMOUZ** [comment]
Should this pr be merged into my other one? Also is there even a way to do so lol

**Comment by jani270** [comment]
> Should this pr be merged into my other one? Also is there even a way to do so lol

For god sake no, the other pr is already a mess with the most useless changes in gods existence. We dont need more shit in there

**Review by jani270 — APPROVED** [review]

**Review by Lulonaut — APPROVED** [review]

**Pull request merged**